src/main/java/frc/utils/time/TimeUtils.java

```diff
@@ -0,0 +1,47 @@
+package frc.utils.time;
+
+import edu.wpi.first.hal.HALUtil;
+import frc.utils.Conversions;
+import frc.utils.alerts.Alert;
+import frc.utils.alerts.AlertManager;
+import frc.utils.alerts.PeriodicAlert;
+import org.littletonrobotics.junction.Logger;
+
+public class TimeUtils {
+
+	public static final double DEFAULT_CYCLE_TIME_SECONDS = 0.02;
```

Comment:
Why isn’t the roborio's cycle time a global constant? It's being used in other files too.

Reply:
Because it related to here, in global there should be constants that don’t have their own class and being used in non related files